__tests__/app.test.js

```diff
@@ -86,7 +86,43 @@ describe('GET /api/articles/:article_id', () => {
       .get('/api/articles/not-a-number')
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('Invalid article ID');
+        expect(body.msg).toBe('Invalid article ID')
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of article objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body
+        expect(Array.isArray(articles)).toBe(true)
+        expect(articles.length).toBeGreaterThan(0)
```

Comment:
Well done for asserting against the length here to ensure the test does not pass by default if we get an empty array back! 😄